clusterdock/utils.py

```diff
@@ -122,33 +124,62 @@ def version_str(version):
         return '.'.join([str(int(x)) for x in version])
 
 
-ADJECTIVES = ['bright', 'new', 'pointed', 'red', 'single', 'brightest', 'big', 'white', 'fixed',
-              'yellow', 'double', 'nearest', 'central', 'lucky', 'famous', 'polar',
-              'particular', 'distant', 'former', 'brilliant', 'pole', 'variable', 'massive',
-              'blue', 'day', 'beautiful', 'evil', 'faint', 'popular', 'female',
-              'biggest', 'golden', 'giant', 'blazing', 'favorite', 'hot', 'lone',
-              'morning', 'fallen', 'north', 'tiny', 'male', 'brittle', 'dark',
-              'solitary', 'top', 'unlucky', 'type', 'typical', 'huge', 'porn',
-              'northern', 'farthest', 'rayed', 'pale', 'nearby', 'green', 'brighter',
-              'musical', 'time', 'dead', 'silent', 'all', 'luminous', 'bunch',
-              'clump', 'flock', 'agglomeration', 'bundle', 'constellate', 'swad', 'huddle',
-              'forgather', 'agglomerate', 'pleiades', 'tuft', 'collective', 'mass', 'component',
-              'fragmentation', 'nucleus', 'heap', 'aggregation', 'clustering', 'tussock', 'knot',
-              'foregather', 'clusters', 'mob', 'supergroup', 'collection', 'multitude', 'bevy',
-              'amass', 'sabha', 'regroup', 'congregate', 'globular', 'uncollected', 'batch',
-              'larger', 'groud', 'components', 'compound', 'gatherer', 'structures', 'semigroup',
-              'cortege', 'convocation', 'concourse', 'gang', 'fragments', 'shells', 'plant',
-              'cells', 'embedded', 'acyl', 'measuring', 'detected', 'markers', 'monoid',
-              'distinct', 'complexes', 'localized', 'scattered', 'congregation', 'craters',
-              'flattened', 'explosive', 'quintet']
+def get_clusterdock_label(cluster_name=None):
+    """
+    Generate a Clusterdock meta data label in json format. Meta data such as: Clusterdock
+    package name, version, location of Clusterdock install -etc.
+
+        Args:
+            cluster_name (:obj:`str`, optional): Cluster name to attach to meta data label.
+                Default: ``None``
+
+        Returns:
+            (json): Clusterdock meta data label
+    """
+    label_str = ''
+    try:
+        package = get_distribution('clusterdock')
+        label_info = {'name': package.project_name, 'version': package.version,
+                      'location': package.location}
+        if cluster_name:
+            label_info['cluster_name'] = cluster_name
+        label_str = json.dumps(label_info)
+    except:
+        # pass
+        raise
```

Comment:
Why even do a try-except if you'll just reraise? In that case, you can just leave out the try-except altogether, no?